plugins/modules/reload.py

```diff
@@ -27,9 +27,24 @@ def run_module():
         target=dict(
             type='str', required=True, aliases=['tgt', 't'],
             choices=[
-                'alias', 'route', 'cron', 'unbound', 'syslog', 'ipsec', 'shaper',
-                'monit', 'wireguard', 'interface_vlan', 'interface_vxlan', 'frr',
-                'webproxy', 'interface_vip', 'bind', 'ipsec_legacy', 'ids',
+                'alias',
+                'rule',
+                'route',
+                'cron',
+                'unbound',
+                'syslog',
+                'ipsec',
+                'ipsec_legacy',
+                'shaper',
+                'monit',
+                'wireguard',
+                'interface_vlan',
+                'interface_vxlan',
+                'interface_vip',
+                'frr',
+                'webproxy',
+                'bind',
+                'ids',
```

Comment:
Its easier to verify that this is consistent with the implementation if its a list in the same order as implementation.